evcxr_repl/src/repl.rs

```diff
@@ -46,9 +54,33 @@ impl Highlighter for EvcxrRustylineHelper {
     fn highlight_prompt<'b, 's: 'b, 'p: 'b>(
         &'s self,
         prompt: &'p str,
-        _default: bool,
+        info: PromptInfo<'_>,
     ) -> Cow<'b, str> {
-        prompt.yellow().to_string().into()
+        if info.default() {
+            if info.line_no() > 0 {
+                self.continuation_prompt.yellow().to_string().into()
+            } else {
+                self.colored_prompt.yellow().to_string().into()
+            }
+        } else {
+            prompt.into()
+        }
+    }
+
+    fn has_continuation_prompt(&self) -> bool {
+        true
+    }
+
+    fn highlight_hint<'h>(&self, hint: &'h str) -> Cow<'h, str> {
+        hint.blue().to_string().into()
```

Comment:
We don't use `hinter` right now. I'd probably prefer waiting until we use it to decide it's styling (blue would be pretty jarring as a autosuggesion color anyway).